coos.html

```diff
@@ -511,49 +523,51 @@ <h3>Base GSBPM model</h3>
 
 				<h3>Adding GAMSO</h3>
 
-				<p>Introducing GAMSO in this framework is not very difficult. The terminology used in the model shows that the main concepts are activities and activity areas. GAMSO describes two kinds of activities: those that are of statistical nature, for example “Manage Statistical Methodology”, and those that are in support of statistical activities, like “Manage Finances”. To account for this distinction, COOS defines two different classes: <code>StatisticalActivity</code> and <code>Activity</code>. <code>Activity</code> is a daughter of <code>prov:Activity</code> and a mother of <code>StatisticalActivity</code>, which in turn is a mother of the <code>StatisticalProductionActivity</code> introduced previously.</p>
+				<p>Introducing GAMSO in this framework is not very difficult. The terminology used in the model shows that the main concepts are activity and activity areas. GAMSO describes two kinds of activities: those that are of statistical nature, for example “Manage Statistical Methodology”, and those that are in support of statistical activities, like “Manage Finances”. To account for this distinction, COOS defines two different classes: <code>StatisticalActivity</code> and <code>Activity</code>. <code>Activity</code> is a daughter of <code>prov:Activity</code> and a mother of <code>StatisticalActivity</code>, which in turn is a mother of the <code>StatisticalProductionActivity</code> introduced previously.</p>
 
 				<p>The GAMSO activity areas are not really activities by themselves, but rather “boxes” used to classify the activities. This makes them instances of <code>skos:Concept</code> to represent this “taxonomic” nature, or more precisely of a specific <code>ActivityArea</code> class, which is a sub-class of <code>skos:Concept</code>.</p>
-
-				<pre class="example nohighlight" title="Examples of GAMSO activity area and activity">
-				&lt;http://id.unece.org/activities/activityArea/1&gt; a coos:ActivityArea ;
-					skos:notation "1" ;
-					skos:prefLabel "Strategy and Leadership"@en .
-
-				&lt;http://id.unece.org/activities/activity/2.3&gt; a coos:Activity ;
-					skos:notation "2.3" ;
-					skos:prefLabel "Monitor Capability Improvements"@en .
-				</pre>
-
+				
 				<p>In addition, COOS defines the <code>OverarchingActivity</code> class, which represents a supporting activity that applies to all production phases, e.g. metadata management, data management, statistical methodology management, etc. These activities correspond to the overarching processes in the GSBPM and the statistical activities in Corporate Support in GAMSO. These activities support the execution of phases and sub-processes effectively. When harmonized and standardized, overarching activities enable the efficient execution of the entire statistical production process.</p>
+				
+				<p>Three individuals of <code>ActivityCategory</code> are define in the ontology to model the types of overarching activities provided by GSBPM</p>
```

Comment:
This paragraph is out of place, it should be part of the previous section on GSBPM...
In addition, I think we should list the three overarching activities in the text and then just show the rdf representation just for Metadata Management.

Reply:
I kept the definition of the OverarchivingActivity in 3.2 but moved the definition of the three gsbpm overaching activity categories and the example to 3.1